### ISSUE TITLE
Bukkit PlotSquared Worlds does not tp player back to lastlogin location upon relogging

### ISSUE THREAD

**Issue by braceyourself07** [IntellectualSites/PlotSquared#1591]
# Bug report template 

**Debug paste link:** 

https://hastebin.com/ezonujigoc

**Description of the problem:** 
When a player logs out of a plotsquared world (worlds: true), when they relog, it does not put the player in the last known location.  The player appears at the plot sethome point.

**How to replicate:** 
Go to a location that is not the plot world's spawnpoint.  Log out.  Log back in, and you will be at the spawnpoint and not the last known location.

**Checklist**:

- [x] I included a `/plot debugpaste` link
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of PlotSquared
- [x] I Made sure the bug/error is not caused by any other plugin

**Comment by dordsor21** [comment]
Are you teleporting to this world via bungee? If so, it is possible bungee is overriding the spawn location to the spawn of the world. I assume this wasn't a problem before P2 world'ing, as MV probably overrode bungee, and teleport the player to the last known location.

**Comment by braceyourself07** [comment]
Hey dords, yeah it's via bungeecord.  Emp and I had adjusted it a few times with PlotSquared and I recall it worked at one point (although it was a double teleport... once by the bungee/server and then again PlotSquared would tp to the last known location) but I noticed that it doesn't do it anymore.

@boy0001 it may have something to do with the adjusted priority when we implemented the redis based warp/tp system.
